packages/rolldown/src/options/input-options.ts

```diff
@@ -17,8 +21,12 @@ export interface InputOptions {
   shimMissingExports?: BindingInputOptions['shimMissingExports']
 }
 
+export type Alias = Omit<AliasItem, 'replacement'> & {
+  replacement: string
+}
+
 export type RolldownResolveOptions = Omit<BindingResolveOptions, 'alias'> & {
-  alias?: Record<string, string>
+  alias?: Record<string, string> | Alias[]
```

Comment:
Please don't change the type of `alias`. For users, they still pass alias using object.

Reply:
Ok, get it